src/services/user.service.js

```diff
@@ -66,7 +66,7 @@ function handleResponse(response) {
       roles = data?.roles;
     }
 
-    if ((!roles.includes('conseiller') && !roles.includes('admin_coop'))) {
+    if ((!roles.includes('conseiller') && !roles.includes('admin_coop') && !roles.includes('structure_coop'))) {
```

Comment:
yes ok ou même en sortant en const les roles autorisés.

```suggestion
const rolesAllowed = ['conseiller', 'admin', 'structure'];
if (rolesAllowed.filter(role => roles.includes(role)).length === 0)
```